README.md

```diff
@@ -1 +1,33 @@
-# dnanexus_verify_bam_id
\ No newline at end of file
+# dnanexus_verifybamid v1.0
+statgen/verifyBamID [v1.1.3] (https://github.com/ewels/MultiQC/releases/tag/v1.3)
+
+## What does this app do?
+This app runs verifyBamID to detect sample contamination from population allele frequencies.
+
+verifyBamID models the sequence reads as mixture of two unknown samples based on the allele frequency information in a provided VCF file. Here, the VCF file used
```

Comment:
"Here, the VCF file used" seems truncated/redundant. VCF is described in next line.

Reply:
Done.